src/reducers/SignIn.js

```diff
@@ -0,0 +1,45 @@
+import {
+  USER_SIGN_IN_FULFILLED,
+  USER_SIGN_IN_FAILED,
+  CLEAR_ERROR,
+  SIGNING_IN_REQUEST
+} from '../actions/action.types';
+
+const initialState = {
+  token: '',
```

Comment:
I don't think we should be saving the `token` to the store. we can always fetch it from localStorage when we need it